tools/pymetasploit3/pymetasploit3/msfrpc.py

```diff
@@ -10,7 +10,7 @@
 import requests.packages.urllib3
 from retry import retry
 
-from pymetasploit3.utils import *
+from pymetasploit3.utils import *  # noqa: F403
```

Comment:
Can you please exclude the folder pytestmettasploit3 instead?

Reply:
done.